src/diagnostic-channel-publishers/src/mysql.pub.ts

```diff
@@ -101,7 +101,7 @@ const mysqlPatchFunction: PatchFunction = function(originalMysql, originalMysqlP
 };
 
 export const mysql: IModulePatcher = {
-    versionSpecifier: ">= 2.0.0 <= 2.14.0",
+    versionSpecifier: ">= 2.0.0 <= 2.14.x",

```

Comment:
Agreed that some kind of automation would be good here. I had initially inspected the code for the 2.0.0 version and saw that there were no changes in the places I had modified between there and 2.13.x, but there could be other code changes that caused issues.